sematic/ui/src/pipelines/RunTree.tsx

```diff
@@ -44,7 +44,7 @@ export default function RunTree(props: {
       }}
     >
       {runTreeNodes.map(({run, children}) => (
-        <Fragment key={run!.id}>
+        <Fragment key={`${run!.id}---${run!.future_state}`}>
```

Comment:
Why the triple dash? Why not `-`?

Reply:
Avoid people who include `-` in UUID values and cause confusion. Eg. in calculating `${A}-${B}`
A = `1-0`
B = `1`

VS 
A = `1`
B= `0-1`

would get same concatenation

It could very rarely happen, but why risk it :)